README.md

```diff
@@ -268,6 +268,10 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // The amount of time before the OIDC Authorization Code grant
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600
+
+                // Name to claim search for to provide mapping of OIDC user
```

Comment:
not sure what this means... "Name to claim search for"

also, how does this relate to our possible future of having alternate mappings?

Reply:
Whoops, issue between thoughts and typing it seems.

I believe this option would only relate to the current user mapping with the OIDC claims. As for 'is this optional' or 'how would this interact if we use mapping B or C', not sure how that would work out.